packages/react-core/src/demos/examples/Nav/NavDefault.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Card,
+  CardBody,
+  Gallery,
+  GalleryItem,
+  Nav,
+  NavItem,
+  NavList,
+  Page,
+  PageSection,
+  PageSectionVariants,
+  PageSidebar,
+  PageSidebarBody,
+  SkipToContent,
+  TextContent,
+  Text
+} from '@patternfly/react-core';
+import { DashboardBreadcrumb } from '../DashboardWrapper';
+import DashboardHeader from '../DashboardHeader';
+
+export const PageLayoutDefaultNav: React.FunctionComponent = () => {
```

Comment:
Just for consistency, let's update these exports in these demos to match the file names. `NavDefault`, etc.